src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const married = (sex, partnerName) => {
```

Comment:
- You can move this function outside the component
- Also, the function name should have a verb, and describe what it does, for example, `getPartnerText`